lib/pbench/agent/base.py

```diff
@@ -35,15 +35,22 @@ def __init__(self, context):
             self.pbench_run = pathlib.Path(self.config.pbench_run)
             if not self.pbench_run:
                 self.pbench_run = pathlib.Path("/var/lib/pbench-agent")
+            try:
+                self.pbench_run.mkdir(exist_ok=True)
+            except Exception as exc:
+                click.secho(
+                    f"[ERROR] unable to create pbench_run directory, '{self.pbench_run}': '{exc}'"
+                )
+                sys.exit(1)
```

Comment:
Rather than invoking `sys.exit()`, consider using `click.get_current_context().exit()` (e.g., like line 32).

(Ditto for lines 55 and 69....)

Reply:
See PR https://github.com/distributed-system-analysis/pbench/pull/2732.